pkg/stabilityai/config/definitions.json

```diff
@@ -28,7 +28,8 @@
             "description": "AI task type",
             "type": "string",
             "enum": [
-              "Text to Image"
+              "Text to Image",
+              "Image to Image"
```

Comment:
@praharshjain and @donch1989, let's support users to create a "Text to Image" task Stability AI connector in this release. 
We will support the "Image to Image" task in the next release, when the pipeline DAG is ready. We don't have to delete related integration code though.

Reply:
removed `Image to Image` for now